kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/core/config/Defaults.kt

```diff
@@ -59,8 +62,7 @@ object Defaults {
    val defaultTestNameCase: TestNameCase = TestNameCase.AsIs
    val specExecutionOrder = SpecExecutionOrder.Lexicographic
 
-   const val concurrentTests = ProjectConfiguration.Sequential
-   const val dispatcherAffinity = true
+//   const val dispatcherAffinity = true
```

Comment:
Intentional?

Reply:
Yes, note the PR comments that this is still tbd while I think about it. I won't merge until it's in a final state.